### DISCUSSION TITLE
livewire in sub directory

### DISCUSSION THREAD

**Discussion by NazarAli** [filamentphp/filament#9775] [General]
i add my filament  app in sub directory in hosting like
http://examle.com/mypp
the problem it not work 
how can change livewire route in application 

![Screenshot 2023-11-20 223756](https://github.com/filamentphp/filament/assets/7056487/d71cd470-eac6-4aed-9e8b-5fffdf30418c)

**Comment by leandrocfe** [comment]
[this](https://livewire.laravel.com/docs/installation#configuring-livewires-update-endpoint)?

**Reply by NazarAli** [reply]
thank bro it work.
but i have another equation in local the minimum request take 3MB  .
but in host take 29MB
same request and same data
